Tests/LibWeb/Text/input/HTML/HTMLSelectElement-can-select-option-in-JS.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
Just to make sure: this test passes without the later changes in this PR, right? If not, please arrange the commits so that it works when added.

Reply:
No, this test will pass only after all three other commits are in. Reordered commits